flowetl/etl/etl/model.py

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Define a DB model for storing the process of ingestion
+"""
+
+import pendulum
+from pendulum.date import Date as pendulumDate
+
+from sqlalchemy import Column, String, DateTime, Date, Integer
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm.session import Session
+
+Base = declarative_base()
+
+# pylint: disable=too-few-public-methods
+class ETLRecord(Base):
+    """
+    DB Model for storing the process of ingestion
+    for found files.
+    """
+
+    __tablename__ = "etl_records"
+    __table_args__ = {"schema": "etl"}
+
+    id = Column(Integer, primary_key=True)
+    file_name = Column(String)
+    cdr_type = Column(String)
+    cdr_date = Column(Date)
+    state = Column(String)
+    timestamp = Column(DateTime)
```

Comment:
This may not apply, but just wondering if you want an additional column for additional free-text metadata such as a comment or debug message? It's quite possible that this is a bad idea though because we want to keep debugging information in a more structured form, so just throwing it out there for consideration.

Reply:
Could put location of the logs for the DAG run in case of a failure? But as it stands I have no envisioned use so will leave it...